classes/plugins/PubIdPlugin.inc.php

```diff
@@ -10,355 +10,271 @@
  * @class PubIdPlugin
  * @ingroup plugins
  *
- * @brief Abstract class for public identifiers plugins
+ * @brief Public identifiers plugins common functions
  */
 
-import('lib.pkp.classes.plugins.Plugin');
+import('lib.pkp.classes.plugins.PKPPubIdPlugin');
 
-abstract class PubIdPlugin extends Plugin {
+abstract class PubIdPlugin extends PKPPubIdPlugin {
 
 	/**
 	 * Constructor
 	 */
 	function PubIdPlugin() {
-		parent::Plugin();
+		parent::PKPPubIdPlugin();
 	}
 
-
 	//
 	// Implement template methods from Plugin
 	//
 	/**
 	 * @copydoc Plugin::register()
 	 */
 	function register($category, $path) {
```

Comment:
No need to include an implementation if all it does is invoke the parent class, I think.